### ISSUE TITLE
Blokada doesn't block ads even though its on and notification doesn't pop out when its enabled.

### ISSUE THREAD

**Issue by BarryThrill** [blokadaorg/blokada#137]
### Problems description
1.  Blokada doesn't blocks ads - Basically I have blacklist from Uni9fied hosts files and also from adblock.mahakala.is which contains 254k hosts.

2. The notification issue - Basically I have set it up as enabled which is pretty much the correct way and even though when I try to go any sites, First of all it doesn't block any ads so I assume meaning it doesn't pop out, Even though there was some sites that got blocked but no notification popped out either.

### Expected behaviour

I assume the issue with blocks would be that it actually blocks those sites that has ads on it, Etc. Twitch. Worked superb on version before the 3.0. Never had a issue and all the ads got blocked, Now everytime I want to watch twitch, it gives me ads, Same thing when it comes to streaming through Chrome app or even normal sites. 

Notification should be tell me a notification telling what hosts it has blocked.

### Actual behaviour

Basically nothing happened with those - No ads getting blocked and no notification popping out at all.

I have tried to restart multiply times, restart and clear cache on blokada, Reinstalled and tried new hosts but still same issue.

### Configuration
1. Blokada version

Latest 3.0.

2. Device model

Google Pixel 2 XL

3. Android version / ROM

Oreo 8.1 stock.

### Questions
1. How long have you been using Blokada? 

Since January

2. How long time ago has the problem started to happen? 

After the 3.0 Update.

3.  How often does this problem happen to you?

It started after the update.

**Comment by ymhuang0808** [comment]
I also encountered the situation. OS is LineageOS 15.1.

**Comment by rpeter85** [comment]
With a quick test we checked if the hosts on the lists are accessible with 'ping' and they are, which gives the impression the host file is not read up or in use.

**Comment by w4tadrag** [comment]
Hi, 

I also want to let you know I am having similar issues with my Amazon Fire 7 tablet. If I am away from the device for few hours or a day and try using it, Blokada doesn't do anything. In the notification section it says monitoring, but does not block ads. The only way for me to make it work was to force stop it and then turn it back on. So it starts working again until I leave the device for few hours or a day. So I have to do the same thing again.

**Comment by rpeter85** [comment]
Can you do the following, please, just to have a log about it:
1.) Ensure that Blokada is running (key icon in the top bar, green shield in Blokada, etc)
2.) Go to a website where ads should be blocked (twitch, any other app)
3.) Collect logcat. Help: https://www.xda-developers.com/guide-sending-a-logcat-to-help-debug-your-favorite-app/
4.) Redact every personal information (IP address, wifi SSID, etc)
5.) Send it. I would like to check what happens in the background

**Comment by kar** [comment]
@BarryThrill could you try to reproduce it on the newest 3.2?

**Comment by kar** [comment]
So far so good, letting it stay a little longer

**Comment by alexanderadam** [comment]
@kar since a few days I have _exactly the same_ behaviour again.
Should I open a new issue for that or rather reopen this one?

It says that it is Blokada 3.4 (100500) (latest version from F-Droid) although the Changelog says that 3.5 (and 4.6 beta) should be available since a while. The app itself also claims that there are no new updates.

"Keep alive" is set to on and "Online only" is set to off since ages because the "online recognition" didn't work reliable for me (it is a different issue though).
And this way it worked until recently.
Putting Blokada off and on again didn't help and device restarts didn't help either.

The app runs on a Galaxy S9 with latest updates  (Android 8.0.0 / Kernel `4.9.59-14049503-QB19624926`). Is there any information I could provide to help investigating into this?

**Comment by kar** [comment]
Wait for 3.6, out soon.

**Comment by alexanderadam** [comment]
Is there any possibility to get 3.6 already?
I could try it because 3.5 isn't working for me anyway.

**Comment by alexanderadam** [comment]
@kar new release works.

But "keep alive" somehow to problems (i.e. #270 and blocking of "everything").  Also enabling and disabling of Blokada didn't seem to work reliable. Those things aren't related to this issue here though so my last comment can be ignored since 3.6.
